default/python/AI/AIDependencies.py

```diff
@@ -299,9 +301,9 @@ class Tags:
 
 EXOBOT_TECH_NAME = "PRO_EXOBOTS"
 
-SHP_WEAPON_ARC_DISRUPTOR_1 = "SHP_WEAPON_ARC_DISRUPTOR_1"
-SHP_WEAPON_ARC_DISRUPTOR_2 = "SHP_WEAPON_ARC_DISRUPTOR_2"
-SHP_WEAPON_ARC_DISRUPTOR_3 = "SHP_WEAPON_ARC_DISRUPTOR_3"
+# SHP_WEAPON_ARC_DISRUPTOR_1 = "SHP_WEAPON_ARC_DISRUPTOR_1"
+# SHP_WEAPON_ARC_DISRUPTOR_2 = "SHP_WEAPON_ARC_DISRUPTOR_2"
+# SHP_WEAPON_ARC_DISRUPTOR_3 = "SHP_WEAPON_ARC_DISRUPTOR_3"
```

Comment:
why uncommenting arc disruptor techs?

Reply:
They are not used anywhere in the code, AFAICT:

```
$ grep -r SHP_WEAPON_ARC_DISRUPTOR default/python/AI/*
default/python/AI/AIDependencies.py:SHP_WEAPON_ARC_DISRUPTOR_1 = "SHP_WEAPON_ARC_DISRUPTOR_1"
default/python/AI/AIDependencies.py:SHP_WEAPON_ARC_DISRUPTOR_2 = "SHP_WEAPON_ARC_DISRUPTOR_2"
default/python/AI/AIDependencies.py:SHP_WEAPON_ARC_DISRUPTOR_3 = "SHP_WEAPON_ARC_DISRUPTOR_3"
default/python/AI/AIDependencies.py:    "SR_ARC_DISRUPTOR": tuple(("SHP_WEAPON_ARC_DISRUPTOR_%d" % i, i) for i in [2, 3]),
default/python/AI/TechsListsAI.py:            "SHP_WEAPON_ARC_DISRUPTOR_1",
```

Code is only using the strings (within quotes), not the definitions (without the quotes).
See how the similar code in AIDependencies.py for SHP_WEAPON_1_1 doesn't require those definitions (do not exist):

```
default/python/AI/AIDependencies.py:    "SR_WEAPON_1_1": tuple(("SHP_WEAPON_1_%d" % i, 1) for i in [2, 3, 4]),
```